src/Tickets/Commerce/Gateways/Stripe/Stripe_Elements.php

```diff
@@ -69,4 +75,22 @@ public function card_element_type() {
 		return tribe_get_option( Settings::$option_checkout_element_card_fields );
 	}
 
+
+	/**
+	 * Are Apple/Google Pay wallets enabled?
+	 *
+	 * @since TBD
+	 *
+	 * @return bool
+	 */
+	public function wallets_enabled() {
+		if ( ! defined( 'EVENT_TICKETS_PLUS_FILE' ) ) {
+			return false;
+		}
```

Comment:
Stephen had shared a map of actions to check per plugin instead of defined. Maybe speak with him.

My opinion though is checking constants its the fastest solution we have in our hands, but if i am not mistaken the constants could be defined but the plugin still not fully loaded. (Min requirements failed for example)

Reply:
@Camwyn Would be very interested to see that list! Thanks

Reply:
I would love to make sur ewe dont use constants, because they can be active when the plugin is soft deactivated, use a `did_action()`

Reply:
[The list for reference](https://lw.slack.com/archives/C01H7Q57P1C/p1725247304646939?thread_ts=1725247070.454189&cid=C01H7Q57P1C)